pkg/apiserver/handlers_test.go

```diff
@@ -151,6 +151,15 @@ func TestGetAPIRequestInfo(t *testing.T) {
 		{"GET", "/namespaces", "list", "", "", "namespaces", "", "Namespace", "", []string{"namespaces"}},
 		{"GET", "/namespaces/other", "get", "", "other", "namespaces", "", "Namespace", "other", []string{"namespaces", "other"}},
```

Comment:
Can you add a test paths for

POST /namespaces
PUT /namespaces/other
PUT /namespaces/other/status
PUT /namespaces/other/finalize

Thanks